service/static/js-src/muk.funder.js

```diff
@@ -1,3 +1,6 @@
+var initialPageLoad = true
+var defaultSize = 10
+
```

Comment:
For all files, this seems to be used in one function, and is effectively a static. 

Would make more sense to move this to the synchronise function, and maybe call it maxIntialRecords